android/src/main/java/com/idehub/GoogleAnalyticsBridge/GoogleAnalyticsBridge.java

```diff
@@ -191,6 +191,16 @@ public void trackSocialInteraction(String trackerId, String network, String acti
     }
 
     @ReactMethod
+    public void setCustomDimensionValue(String trackerId, Integer dimensionIndex, String dimensionValue)
+    {
+        Tracker tracker = getTracker(trackerId);
+
+        if (tracker != null) {
+          tracker.set(Fields.customDimension(dimensionIndex), dimensionValue);
```

Comment:
This implementation is incorrect. The bit with `set(...)` and `Fields.customDimension(..)` is from the v3 Android GA library. I am using v4 for this library, [docs here](https://developers.google.com/analytics/devguides/collection/ios/v3/customdimsmets#set-send).
Also it seems that you have to send the dimension along with a screen view or event.